update(UI overview): add bonita apps (#1362)
bonitasoft/bonita-doc: md/processes.md

3. Click on _More_ button.
4. A summary of configuration problems is displayed below the process name in a well and warning icons are displayed in the left navigation list.
5. Modify your configuration for each element displayed.

To resolve a process for business variables:
1. Logout
2. Login with the [technical user credentials](tenant_admin_credentials.md)
3. [Pause the BPM services](pause-and-resume-bpm-services.md)
4. [Update the BDM](bdm-management-in-bonita-bpm-portal.md)
5. Come back to _BPM_/_Processes_ and check the status.

## Enable a process
1. Go to _BPM_>_Processes_ (Administrators) or _Processes_ (Process Managers)
2. Click on the _Resolved_ filter
3. Select the process
4. Click on the _More_ button
5. Click on the _Enable_ button
6. The activation state changes to "enabled"
7. The process disappears from the disabled filter and is now displayed in the enabled filter

::: info
**Note:** You can also select the processes you want to enable by checking the box in the _Resolved_ list, then click on the _ENABLE_ button in the bulk actions section, at the top of the list.  

> You can also enable a process in the _More details_ view, by clicking on the _Enable_ button.

Reviewer: isn't that method just what you explained in the steps above ? I think you need to remove this sentence or change the steps to remove step 5 and remove the first sentence of the note.
In Disable a process, the bulk action is used in the steps so it should be consistent.

Author: oui oui oui

**Warning:** **Beware! Data loss risk!**
Deleting a process will automatically delete all its cases (on-going and archived alike). Thus, the operation may take a long time, and fail if the transaction timeout is not large enough.
This feature should only be used on non-production environments.
**Proceed at your own risk.**
:::

1. Go to _BPM_>_Processes_ (Administrators) or _Processes_ (Process Managers),
2. Select the process to delete by checking the tickbox next to the process
3. Click on the _Delete_ button

> 

Reviewer: there is now a confirmation modal as well. Maybe we can add a step ?

Author: ok

2. Select a process in the list
3. Click on the _More_ button
4. In _General_, click the _Pencil_ next to _Categories_ label
5. In the opened modal window, start typing the category name, and then select among the suggestions by using the arrow keys or mouse.
6. Press the _**Enter**_ key
7. Click on the _Save_ button

## Start a case for another user
This feature is available with the Enterprise, Performance, and Efficiency editions.  
To start a case for another user:
1. Go to _BPM_>_Processes_ (Administrators) or _Processes_ (Process Managers)
2. Select the process and click on the _Start for_ button.
3. In the popup, specify the user for whom you are starting the case. Only valid users for the case are displayed.
4. Click on _Start_.

The case is started as though the specified user had started it. 
For example, if a you start a case for user A and a subsequent task is to be done by the manager of the user, it is assigned to user A's manager, not to your manager.

All of what follows belong to the [Live update feature](live-update.md) and are only available in for the Enterprise, Performance, and Efficiency editions.

## Edit the actor mapping

> You need to map more or less organization entities to an actor. To do so:

Reviewer: why "more or less" ? this is exactly that.
```suggestion
You need to map organization entities to an actor. To do so:
```

1. Go to _BPM_>_Processes_ (Administrators) or _Processes_ (Process Managers)
2. Select a process in the list.
3. Click on the _More_ button.
4. Click on _Actors_ in the left menu.
To add entities:
5. Click the _+_ button in the user, group, role or membership column of the actor line.

> 6. In the opened modal window, click on the dropdown list to select one or several actors. The list displays the first five elements, then a number is displayed representing the other selected entities.

Reviewer: ```suggestion
6. In the opened modal window, click on the dropdown list to select one or several entities. The list displays the first five elements, then a number is displayed representing the other selected entities.
```

To remove entities:

> 6. In the opened modal window, there is a list of the actors already mapped. Click on the _X_ button next to the actor, or click on _Remove all_. A list appears filled with the actors you are about to remove. 

Reviewer: ```suggestion
7. In the opened modal window, there is a list of the entities already mapped with the actor. Click on the _X_ button next to an entity, or click on _Remove all_. A list appears filled with the entities you are about to remove. 
```

>    You can **undo** a removal by clicking on the _X_ button next to the actor or by clicking _Enable all_ 

Reviewer: entity
```suggestion
   You can **undo** a removal by clicking on the _X_ button next to the entity or by clicking _Enable all_ 
```

There are two stages to define a Process Manager:

1. Map organization entities to access the _Process Manager_ Portal to view information related to the processes they will be _Process Managers_ of.
   Only Administrators can do so.
    1. Go to _Organization_/_Profiles_
    2. Choose the _Process manager_ profile. This will display the users, groups, roles, and memberships who already have this profile.
    3. Click on _More..._.
    4. Click on _Add a user_.
    5. Select a user by checking the box next to the user name.
    6. Click on _Add_. 
     The _Process manager_ profile is now assigned to the user.
     You can also assign a profile to a group, role, or membership.
     
2. Map the process to one or several entities that have the _Process Manager_ profile. 
    1. Go to _BPM_>_Processes_ (Administrators) or _Processes_ (Process Managers) 
    2. Select the process 
    3. Click on _More..._. The Process managers currently assigned to the process are shown in the _Process manager mapping_ table.
    4. To add a user, click the _Plus_ sign or _Pencil_ button in the _User_ column. A popup shows a list of the users who are currently process managers for this process.

>     5. Click _Select users_. A list of users with the Process manager profile is displayed.

Reviewer: No, it is not filtered. It displays all the users
```suggestion
    5. Click _Select users_. A list of users is displayed.
```

3. Click on _Connectors_ in the left menu
4. In the connector definitions table, in the actions column, click on the _Pencil_ icon
5. Browse to a .zip file containing the new connector implementation
6. Click on _Save_ to import the new implementation.

## Upload a new form
1. Go to _BPM_>_Processes_
2. Click on the _More_ button
3. Click on _Forms_ in the left menu
4. Click on the _Form list_ tab. A list of form is displayed. These forms are only visible to the current process. 
5. Click the _plus_ button at the bottom of the list. A file selector popup is displayed. 
6. Browse to a .zip containing a form.
7. Click on _Next_, then if your form requires some authorization, it will be displayed.
8. Click on _Confirm_.

## Edit an existing form
1. Go to _BPM_>_Processes_
2. Click on the _More_ button
3. Click on _Forms_ in the left menu
4. Click on the _Form list_ tab. A list of forms is displayed. These forms are only visible to the current process.
5. Click the _Pencil_ button of a form line. A file selector popup is displayed.
6. Browse to a .zip containing a form
7. Click _Next_, then if your form requires some authorization, it will be displayed.
8. Click on _Confirm_.

> ## Fix forms

Reviewer: ```suggestion
## Change the instantiation or tasks forms or the overview page
```

1. Go to _BPM_>_Processes_
2. Click on the _More_ button
3. Click on _Forms_ in the left menu.
4. The tables are displayed which list available forms. Click on a red link. A field input appears.
5. Starting to type some text, and any matching names of installed forms will be proposed. If there is no match for the name you enter, it will be considered as a URL.
6. Click the _Tick_ button to validate your change or _**X**_ to dismiss your change.

::: info

> **Note:** You can also upload a form in order to create a new mapping.

Reviewer: ```suggestion
**Note:** You can also upload a new form in order to create a new form mapping (see the above section Upload a new form).
```
